sof-js/src/path.js

```diff
@@ -72,7 +181,7 @@ function process_constants(constants) {
 }
 
 export function fhirpath_evaluate(data, path, constants = []) {
-  return fhirpath.evaluate(data, rewrite_path(path), process_constants(constants), null, fhirpath_options);
+  return fhirpath.evaluate(data, rewrite_path(path), process_constants(constants), fhir_r4_model, fhirpath_options);
```

Comment:
Note: if we don't use model, TypeInfo for Resources will not work. 

Question: should we use R4 in reference?

Reply:
I guess we can. Do we need it to tell date and dateTime apart?

Reply:
Yes, to determine date and dateTime, it is important to use the model